internal/istio/istiofeature/reconcile_remoteistio_cr.go

```diff
@@ -155,9 +149,5 @@ func (m *MeshReconciler) generateRemoteIstioCR(config Config, ipRanges *pkgClust
 		},
 	}
 
-	if config.BypassEgressTraffic {
```

Comment:
Why don't you use something like this here?
```
	if config.BypassEgressTraffic {
		istioConfig.Spec.OutboundTrafficPolicy = v1beta1.OutboundTrafficPolicyConfiguration{
			Mode: "ALLOW_ANY",
		}
	} else {
		istioConfig.Spec.OutboundTrafficPolicy = v1beta1.OutboundTrafficPolicyConfiguration{
			Mode: "REGISTRY_ONLY",
		}
	}
```

Reply:
The `RemoteIstio` doesn't have those options in the spec.

Reply:
Ah, makes sense then :) Thanks!